include/faker-cxx/LuhnCheck.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <string>
+#include <algorithm>
+
+namespace faker
+{
+class LuhnCheck
+{
+public:
+
+    /**
+     * @brief Returns luhn checksum value for the given value.
+     *
+     * @param inputString The string to calculate the check value for.
+     *
+     * @return The luhn checksum value for the given value.
+     *
+     */
+    static int luhnCheckSum(const std::string& inputString) {
```

Comment:
please move it to src/common (its not the faker api) and create LuhnCheck.cpp file with implementations

Reply:
Done